### ISSUE TITLE
Add documentation

### ISSUE THREAD

**Issue by RVxLab** [webparking/laravel-cash#18]
I suggest creating a documentation page of sorts, be it in markdown or maybe using the wiki functionality within GitHub.

The Cash documentation itself is terrible to be frank.

**Comment by remkobrenters** [comment]
Agree. Maybe also add documentation for some scenario's (fetching index, filtering index, getting one record, inserting a record, updating a record and other relevant cases).
